packages-exp/auth-exp/src/core/errors.ts

```diff
@@ -433,3 +431,112 @@ export const _DEFAULT_AUTH_ERROR_FACTORY = new ErrorFactory<
   AuthErrorCode,
   AuthErrorParams
 >('auth', 'Firebase', _prodErrorMap());
+
+/**
+ * A map of potential Auth error codes, for easier comparison with errors
+ * thrown by the SDK.
+ *
+ * @remarks
+ * Please note that you can't tree-shake individual keys
+ * in the map, so by using the map you might substantially increase your
+ * bundle size.
+ *
+ * @public
+ */
+export const AUTH_ERROR_CODES_MAP_DO_NOT_USE_INTERNALLY = {
```

Comment:
I think we want to use `as const` to prevent type widening, so we show the actual literal string values in the reference docs, and people get better intellisense (the literal string vs string) in IDE.

Reply:
Done.